doc/related_projects.rst

```diff
@@ -207,6 +207,23 @@ Other packages useful for data analysis and machine learning.
 - `Deep Learning <http://deeplearning.net/software_links/>`_ A curated list of deep learning
   software libraries.
 
+Recommendation Engine packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
hmm, it seems like the other subgroups use bold text (e.g. https://github.com/scikit-learn/scikit-learn/pull/8464/files#diff-dbb60e1ab48e513230977052f947a772L160) instead of an rst subsection